templates/federated_reporting/import.sh

```diff
@@ -24,10 +24,19 @@ fi
 
 # TODO: we should do some validation of the files here
 mkdir -p "$CFE_FR_SUPERHUB_IMPORT_DIR"
-mv "$CFE_FR_SUPERHUB_DROP_DIR/"*".sql.$CFE_FR_COMPRESSOR_EXT" "$CFE_FR_SUPERHUB_IMPORT_DIR" ||
-  {
-    log "No files in drop dir."
-  }      
+no_drop_files=0
+ls -l "$CFE_FR_SUPERHUB_DROP_DIR/"*".sql.$CFE_FR_COMPRESSOR_EXT" 2>/dev/null ||
```

Comment:
This should also do `>/dev/null`, right?

Reply:
ack.